### ISSUE TITLE
Help with shapes

### ISSUE THREAD

**Issue by davidrahd** [viblo/pymunk#135]
I'm making a simple shooter and I'm having issues with the shape position.
[my game window](https://ibb.co/hFvJHH)
As you can see in the screenshot my shapes are perfect centered with the asteroids, but the shape for my ship is not. my code is spreed in some modules but basically it is:

> image = pyglet.resource.image('ship.png')
> 		image.anchor_x = image.width // 2
> 		image.anchor_y = image.height // 2
> 
> vertices = [[(-10, 10), (-46, -10), (-45, -20), (-13, -26), (13, -26), (45, -20), (46, -10), (10, 10)], [(-6, 35), (-10, 10), (10, 10), (6, 35)], [(-13, -26), (-6, -35), (6, -35), (13,-26)]]
> 	    
> self.body = pymunk.Body(mass, moment, body_type)
> self.body.angle  = math.radians(angle)
> self.body.center_of_gravity = center_of_gravity
> self.body.position = position
> self.body.velocity = velocity
> self.body.angular_velocity = angular_velocity
> self.space.add(self.body)
> 
> self.sprite = pyglet.sprite.Sprite(img=image, batch=player_batch, subpixel=True)
> self.sprite.position = self.body.position
> self.sprite.rotation = math.degrees(-self.body.angle)
> 
> for vertices in self.vertices :
>                                 poly = pymunk.Poly(body=self.body, vertices=vertices, radius=1)
> 				self.shapes.append(poly)
> 				self.space.add(poly)

My asteroids have the same code, the only difference they have only one shape, but even when I try to use only one of my ship's shapes it also appears off centered. I got my ship's coordinates the same way I did with the asteroids, using Gimp's tool and measuring the distance from the center to the point.
Tried the same code using python 32 and 64 bits, 3.1, 3.3 and 3.6, on windows and linux.
I'm doing something wrong? Sorry if it is a noob question to post here in the GitHub, i didn't find a pymunk forum, I'm a mathematician not a programmer, maybe I'm missing something.

**Comment by davidrahd** [comment]
Sorry i'm stupid it was a issue in how i structured my modules

**Comment by viblo** [comment]
no worries :)

**Comment by davidrahd** [comment]
it would be nice to have a forum for less technical stuff where people could help witch other and share their projects. Anyway thanks for the good work porting this amazing physics engine for python.

**Comment by viblo** [comment]
Sadly I dont think the community of Pymunk users is big enough to motivate a whole forum, its would very likely be mostly dead. Previously I have been recommending people to the Chipmunk forum (http://chipmunk-physics.net/forum/). The overall structure is the same in Pymunk and Chipmunk so many questions are similar, and also for sharing projects. However, I know that the Chipmunk forum is not very active anymore.

Basically Im open for suggestions where to point users to, but it has to be somewhere that doesnt feel too dead.
